docs/application/dotnet/samples/account/overview.md

```diff
@@ -63,6 +63,13 @@ for TD:
 </div>
 
 <div id="TV" class="tabcontent">
+<table>
+	<tbody>
+		<tr>
+			<td>There is no sample.</td>
```

Comment:
What is the context, I am not able to comprehend clearly the usage of "There is no sample" 

Rather, should it be like this?

**Note:** Currently, there are no samples available for the accounts module.
(This note will follow after Accounts description.)

and not like this  
<table>
<tbody>
<tr>
<td>sample image.jpg</td>
<td><strong> Accounts</strong><br /> Description............................................................</td>
</tr>
</tbody>
</table>
<table>
 	<tbody>
 		<tr>
 			<td>There is no sample.</td>
 		</tr>
 	</tbody>
 </table>

Reply:
Sorry for late answer, I am off day

Each category has three tabs - Mobile / Wearable / TV...
If there is no sample application for Wearable profile, just empty page displays..

This PR suggests to use some message for this like "There is no sample"...

but, How about to use positive wording like "Samples will be added"...

Reply:
@nawab-samsung
Thank you for your review but I hope you leave a message after understand what this PR is.
Could you see https://docs.stg.tizen.org/staging/504/application/dotnet/samples/account/overview?  The markdown files having HTML tags are not correctly displayed on GitHub with a web browser. 

For the first time, I was supposed to leave a message like "We will add samples soon" but we don't know exactly when a new sample is added.

If a sample will be added, the table format is matched with others.

I'm going to merge this PR soon.

Reply:
@journey2w  thank you for making things lucid! 
- I understand the fact that GITHUB uses Git flavored Markdown only anything out of that does not render properly 
- Since links won't ever work within GIT pages as same are resolved while conversion to .html happens.
- I would have definitely understood the PR more clearly if and else (PR staging link) was provided. 

Isn't it good to entice readers with an anticipation message, rather than pure negation? 
Instead of There is no sample. 
Currently, there are no samples available for this profile.

Reply:
@annie-samsung  and @nawab-samsung
SRI-B review is extremely valuable.  Some of SRI members, however, think they should change some sentences on PRs to show what they review. I hope they should've done. 

If you have better expression to understand, please leave a comment.